_posts/en/meetings/2017-02-16-meeting.md

```diff
@@ -0,0 +1,272 @@
+---
+title: IRC meeting summary for 2017-02-16
+permalink: /en/meetings/2017/02/16/
+name: 2017-02-16-meeting
+type: meetings
+layout: page
+lang: en
+version: 1
+---
+{% include _toc.html %}
+{% include _references.md %}
+ 
+- [Link to this week logs](https://botbot.me/freenode/bitcoin-core-dev/2017-02-16/?msg=81092667&page=3)
+- [Meeting minutes by meetbot](http://www.erisian.com.au/meetbot/bitcoin-core-dev/2017/bitcoin-core-dev.2017-02-16-19.02.html)
+ 
+---
+
+## Main topics
+
+- 0.14.0RC1 release
+- Randomness
+- Clocks
+
+## 0.14.0RC1 release
+
+### Background
+
+Bitcoin Core 0.14.0 is [scheduled][#8719] to be released around March 2017. Open pull request aimed for 0.14 are [tagged with a 0.14 tag](https://github.com/bitcoin/bitcoin/pulls?q=is%3Aopen+is%3Apr+milestone%3A0.14.0).
+
+### Meeting comments
+
+General sentiment was that release readiness has improved significantly
```

Comment:
please do not break lines manually, if the pages get translated, this is an obstacle.